src/plugins/kibana/public/management/styles/main.less

```diff
@@ -41,6 +41,30 @@ kbn-management-app {
 
 kbn-management-landing {
   padding: 5px;
+}
+
+  .management-panel-product {
+    margin-bottom: 10px;
+
+    .panel-body {
```

Comment:
In this case I think it makes sense to let `panel-body` remain nested since it's closely coupled with the idea of the `management-panel-product`.